node/risk-app/server/models/search-view.js

```diff
@@ -26,12 +26,10 @@ class SearchViewModel {
       items
     }
 
-    if (warnings && warnings.message) {
-      if (warnings.severity && warnings.severity < 4) {
-        this.banner = {
-          url: floodWarningsUrl + '/location?q=' + encodeURIComponent(postcode),
-          message: warnings.message
-        }
+    if (warnings?.message && warnings?.severity < 4) {
```

Comment:
I won't take credit as I can see SonarCloud has raised this as a magic number.

Reply:
SonarCloud raised the magic number issue about a HTTP result in a TODO test I put in, that needs to be done when we move those tests to Jest.
I'm tempted to put a comment in here instead, as it would be more useful than a const